subprojects/scala/src/test/groovy/org/gradle/api/plugins/scala/ScalaPlugin3Test.groovy

```diff
@@ -1,62 +0,0 @@
-/*
- * Copyright 2009 the original author or authors.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package org.gradle.api.plugins.scala
-
-import org.gradle.api.Project
-import org.gradle.api.file.FileCollectionMatchers
-import org.gradle.api.plugins.JavaPlugin
-import org.gradle.api.reporting.ReportingExtension
-import org.gradle.api.tasks.scala.ScalaDoc
-import org.gradle.test.fixtures.file.LeaksFileHandles
-import org.gradle.test.fixtures.file.TestNameTestDirectoryProvider
-import org.gradle.test.fixtures.Flaky
-import org.gradle.util.TestUtil
-import org.junit.Rule
-import org.junit.Test
-import org.junit.experimental.categories.Category
-import spock.lang.Issue
-
-import static org.gradle.api.tasks.TaskDependencyMatchers.dependsOn
-import static org.hamcrest.CoreMatchers.equalTo
-import static org.hamcrest.CoreMatchers.instanceOf
-import static org.hamcrest.MatcherAssert.assertThat
-
-@Issue("https://github.com/gradle/gradle-private/issues/3440")
-@Category(Flaky)
-class ScalaPlugin3Test {
-    @Rule
-    public TestNameTestDirectoryProvider temporaryFolder = new TestNameTestDirectoryProvider(getClass())
-
-    @LeaksFileHandles
```

Comment:
Now that the `@LeaksFileHandles` is gone, I suppose we need to edit the build file as well and make this strict again, right?

Reply:
I do not understand the question, but I believe I misunderstood the `@LeaksFileHandles` initially.
My expectation is that fixing the test setup should no longer leak anything.

Reply:
Yes, you are right. Removing the `@LeaksFileHandles` should be enough.